tests/core/variant/test_array.h

```diff
@@ -129,6 +129,22 @@ TEST_CASE("[Array] has() and count()") {
 	CHECK(arr.count(2) == 0);
 }
 
+TEST_CASE("[Array] find_custom() and rfind_custom()") {
+	Array arr;
+	arr.push_back(1);
+	arr.push_back(2);
+	arr.push_back(3);
+	arr.push_back(4);
+
+	CHECK(arr.find_custom(callable_mp(&arr, array_test_callable)) == 1) // Should be the number 2
```

Comment:
```suggestion
	CHECK(arr.find_custom(callable_mp_static(array_test_callable)) == 1) // Should be the number 2.
```
Fix for callable, and comment